package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "htmlhint",
-    "version": "0.9.14",
+    "version": "0.0.0-development",
```

Comment:
Won't this confuse users if upcoming versions happen to be lower than the existing ones? Or am I misunderstanding how this will work?